token/js/flow-typed/npm/@solana/web3.js_vx.x.x.js

```diff
@@ -120,6 +120,67 @@ declare module '@solana/web3.js' {
     secretKey: Buffer;
   }
 
+  /**
+   * Keypair signer interface
+   */
+  declare export interface Signer {
+    get publicKey(): PublicKey;
+    get secretKey(): Uint8Array;
+  }
+
+  /**
+   * An account keypair used for signing transactions.
+   */
+  declare export class Keypair {
+    /**
+     * @internal
+     *
+     * Create a new keypair instance from a {@link SignKeyPair}.
+     *
+     * @param keypair ed25519 keypair
+     */
+    constructor(keypair: SignKeyPair): this;
```

Comment:
@jstarry any pointers on surfacing `SignKeyPair` for `flow-typed`?  I tried to add stubs [here](https://github.com/solana-labs/solana-program-library/pull/1765/commits/0687cdaeb9cad0885f567e31545ffe25d10925e8#diff-fa7f2645be2837bf96e706e4c01436cc64ae419efd818846d2a4b8ebf7edc126R17), but no dice

Reply:
I recommend dropping this constructor from the flow type to avoid this issue entirely

Reply:
> I recommend dropping this constructor from the flow type to avoid this issue entirely

I like this option :slightly_smiling_face: